common/lib/common-utils/src/bufferBrowser.ts

```diff
@@ -59,7 +62,7 @@ export const bufferToString = (blob: ArrayBufferLike, encoding: string): string
  */
 export function isArrayBuffer(obj: any): obj is ArrayBuffer {
     const maybe = obj as (Partial<ArrayBuffer> & Partial<Uint8Array>) | undefined;
-    return obj instanceof ArrayBuffer
+    return is.ArrayBuffer(obj)
```

Comment:
@vladsud -- finally seeing the forest through the trees here -- is this method intended to validate that the passed `obj` is a genuine `ArrayBuffer` (possibly across iframes), and thus the subsequent "duck-type-check" can be removed or should the criteria stay more loose?